From 41ed20b9703027dfd7c644099bdbe7a9de3775f5 Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sat, 5 Feb 2022 19:54:40 -0500
Subject: [PATCH 2/2] Relax dependency on bindgen

Forwarded: not-needed

This dependency is designed by upstream to keep their MSRV low, but
debian doesn't need to worry about that (the distro's MSRV is managed
differently).
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 07dc4f0..f935b7f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,7 +27,7 @@ dependencies = ["pkg-config", "nettle-dev"]
 [dependencies.libc]
 version = "0.2"
 [build-dependencies.bindgen]
-version = ">= 0.53.1, < 0.58.0"
+version = ">= 0.53.1, < 0.60.0"
 features = ["runtime"]
 default-features = false
 
-- 
2.34.1

